### PR TITLE
Edit CreateWorkTimeConsumer

### PR DESCRIPTION
**Pull request by SolovyovDmitry**
Part of Task Feature #1051

### CODE REVIEW

**Review by ndoriomedov — APPROVED** [review]

File: src/TimeService.Broker/Consumers/CreateWorkTimeConsumer.cs [hunk 1]
```diff
@@ -1,47 +1,47 @@
...
-                new DbWorkTime
-                {
-                    Id = Guid.NewGuid(),
-                    ProjectId = request.ProjectId,
-                    UserId = userId,
-                    Month = timeNow.Month,
-                    Year = timeNow.Year
-                });
-            }
+      foreach (Guid userId in publish.UserIds)
```
Comment (MariyaSin) [inline src/TimeService.Broker/Consumers/CreateWorkTimeConsumer.cs:19]: remove spase

File: src/TimeService.Broker/Consumers/CreateWorkTimeConsumer.cs [hunk 2]
```diff
@@ -1,47 +1,47 @@
...
+      {
+        await _workTimeRepository.CreateAsync(
```
Comment (MariyaSin) [inline src/TimeService.Broker/Consumers/CreateWorkTimeConsumer.cs:21]: this is a bad way to call the repository in a loop

File: src/TimeService.Broker/Consumers/CreateWorkTimeConsumer.cs [hunk 3]
```diff
@@ -1,47 +1,47 @@
...
-        public async Task Consume(ConsumeContext<ICreateWorkTimeRequest> context)
-        {
-            object result = OperationResultWrapper.CreateResponse(CreateWorkTime, context.Message);
+    public async Task Consume(ConsumeContext<ICreateWorkTimePublish> context)
+    {
+      object result = OperationResultWrapper.CreateResponse(CreateWorkTime, context.Message);
 
-            await context.RespondAsync<IOperationResult<bool>>(result);
-        }
+      await context.RespondAsync<IOperationResult<bool>>(result);
```
Comment (MariyaSin) [inline src/TimeService.Broker/Consumers/CreateWorkTimeConsumer.cs:44]: this is publish request, remove respond

File: src/TimeService.Data/WorkTimeRepository.cs [hunk 4]
```diff
@@ -39,6 +39,25 @@ public class WorkTimeRepository : IWorkTimeRepository
...
+    public async Task CreateAsync(List<Guid> userIds, Guid projectId)
+    {
+      if (userIds == null)
+      {
+        return;
+      }
+
+      DateTime timeNow = DateTime.Now;
+      _provider.WorkTimes.AddRange(userIds.Select(u => new DbWorkTime
+      {
```
Comment (MariyaSin) [inline src/TimeService.Data/WorkTimeRepository.cs:51]: map in consumer ar create mapper, it is bad idea to map in repo

**Review by MariyaSin — APPROVED** [review]

**Pull request merged**